.circleci/config.yml

```diff
@@ -153,8 +153,11 @@ commands:
       - run:
           name: Deploying to k8s cluster for service << parameters.k8s_svc_name >>
           command: |
+            git clone https://github.com/binary-com/devops-ci-scripts
```

Comment:
Nitpick: In the future, we should instead have this in the image we're running in circle ci.
Whenever we do anything like this in the middle of a build, we're stalling circle ci, and delaying every build we run.

That's a note for the future though.

Reply:
Yes, I agree with that